src/GoodsList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+  type Props = {
+    goods: string[];
+  };
+
+export const GoodsList: React.FC<Props> = ({ goods }) => {
+  return (
+    <ul>
+      {goods.map((good) => (
+        <li key={good}>
+          {good}
+        </li>
+      ))}
+    </ul>
+  );
+};
```

Comment:
you may use shorthand return syntax here
```suggestion
export const GoodsList: React.FC<Props> = ({ goods }) => (
    <ul>
      {goods.map((good) => (
        <li key={good}>
          {good}
        </li>
      ))}
    </ul>
  );
```